mlflow/utils/uri.py

```diff
@@ -24,6 +24,10 @@ def is_local_uri(uri):
     if uri == "databricks":
         return False
 
+    if is_windows() and uri.startswith("\\"):
```

Comment:
`\a\b\c` is a

https://learn.microsoft.com/en-us/dotnet/standard/io/file-path-formats:

> 

\Program Files\Custom Utilities\StringFinder.exe | A relative path from the root of the current drive.
-- | --

Reply:
ah, good catch!